scim2-sdk-common/src/test/java/com/unboundid/scim2/common/UserResourceVarargsTestCase.java

```diff
@@ -0,0 +1,563 @@
+/*
```

Comment:
This test is very long, and just repeats for every MVA on a SCIM user. It's repetitive and verbose, but I wanted to be sure that a change to just one of these methods didn't break any of the contracts that we're expecting.